presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1097,6 +1098,11 @@ public SystemSessionProperties(
                         "Enable query optimization with materialized view",
                         featuresConfig.isQueryOptimizationWithMaterializedViewEnabled(),
                         true),
+                booleanProperty(
+                        ENABLE_DISTRIBUTED_TRACING,
+                        "Enable distributed tracing of the query",
+                        false,
+                        false),
```

Comment:
Let's add the flag in TracingConfig and set the default based on the config value

Reply:
Yes, what I mean is to add a flag in TracingConfig with `tracing.tracer-enable`